### PR TITLE
Release Solidus 4.1 (version + blog post)

### PR DESCRIPTION
**Pull request by kennyadsl**
## Summary

Add the new version to the list and the blog post (waiting for the cover image, cc @mfrecchiami ❤️ ).

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [ ] I have written a thorough PR description.
- [ ] I have kept my commits small and atomic.
- [ ] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

The following are not always needed:

- 📖 I have updated the README to account for my changes.
- 📑 I have documented new code [with YARD](https://www.rubydoc.info/gems/yard/file/docs/Tags.md).
- 🛣️ I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- ✅ I have added automated tests to cover my changes.
- 📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

File: source/blog/2023-06-27-solidus-v4-1.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Solidus v4.1
+date: 2023-05-08
```
Comment (waiting-for-dev) [inline source/blog/2023-06-27-solidus-v4-1.html.erb:3]:
```suggestion
date: 2023-06-27
```

File: source/blog/2023-06-27-solidus-v4-1.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+tags: Solidus
+author: Solidus Core Team
+cover_image: /blog/2023/05/08/solidus-v4-0/cover.jpg
```
Comment (waiting-for-dev) [inline source/blog/2023-06-27-solidus-v4-1.html.erb:6]:
```suggestion
cover_image: /blog/2023/06/27/solidus-v4-0/cover.jpg
```

File: source/blog/2023-06-27-solidus-v4-1.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Solidus v4.1 is the first minor release of the 4.x series. It contains a number of new features, bug fixes, and performance improvements.
+
+This release is relatively tiny and should be quite easy to upgrade. It starts to lay down the foundational
+work for the new Admin, which will be delivered gradually during the rest of the v4.x releases.
+
+In fact, with v4.1, stores can switch their Admin to the new color scheme, and allow operators to start
+getting used to it, reducing the impact of future changes.
+
+But there's much more. Please read on for the full list of changes in the official [v4.1 Changelog](https://github.com/solidusio/solidus/releases/tag/v4.0.0)
```
Comment (waiting-for-dev) [inline source/blog/2023-06-27-solidus-v4-1.html.erb:20]:
```suggestion
But there's much more. Please read on for the full list of changes in the official [v4.1 Changelog](https://github.com/solidusio/solidus/releases/tag/v4.1.0)
```

**Review by waiting-for-dev — COMMENTED** [review]
Thanks!! Dates and link to the tag need to be fixed, but the content is great.

File: source/blog/2023-06-27-solidus-v4-1.html.erb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+cover_image: /blog/2023/05/08/solidus-v4-0/cover.jpg
+---
+
+- <a class="button" href="https://github.com/solidusio/solidus/releases/tag/v4.1.0">4.1.0 on GitHub</a>
+- <a class="button button-secondary" href="https://rubygems.org/gems/solidus/versions/4.1.0">4.1.0 on RubyGems</a>
+
+Solidus v4.1 is the first minor release of the 4.x series. It contains a number of new features, bug fixes, and performance improvements.
+
+This release is relatively tiny and should be quite easy to upgrade. It starts to lay down the foundational
+work for the new Admin, which will be delivered gradually during the rest of the v4.x releases.
```
Comment (elia) [inline source/blog/2023-06-27-solidus-v4-1.html.erb:15]:
```suggestion
work for the new Admin, which will be delivered gradually starting from one of the v4.x releases.
```

We could easily release a major before the new admin is officially delivered and fully available.

**Review by elia — COMMENTED** [review]
Left a suggestion that reduces commitment around the release of the new admin

**Comment by kennyadsl** [comment]
@elia @waiting-for-dev all your comments should be addressed now. I also added the final cover image.

**Review by rainerdema — APPROVED** [review]

**Pull request merged**